test/seneca.test.js

```diff
@@ -1170,6 +1170,7 @@ describe('seneca', function () {
     si.options({errhandler: done})
 
     si.use('entity')
+    si.use('mem-store')
 
```

Comment:
Working on it now. I need to release another version of entity before I can remove that. I wanted to rebase your tests in first. Once I up the version of entity I can drop that reference and run it against some of the actual stores to verify it's all good 👍  This won't be going in until fully checked.